pkg/operator/csidriveroperator/csioperatorclient/ibm-vpc-block.go

```diff
@@ -0,0 +1,40 @@
+package csioperatorclient
+
+import (
+	"os"
+	"strings"
+
+	configv1 "github.com/openshift/api/config/v1"
+)
+
+const (
+	IBMVPCBlockCSIDriverName          = "vpc.block.csi.ibm.io"
+	envIBMVPCBlockDriverOperatorImage = "IBM_VPC_BLOCK_DRIVER_OPERATOR_IMAGE"
+	envIBMVPCBlockDriverImage         = "IBM_VPC_BLOCK_DRIVER_IMAGE"
+	envIBMVPCNodeLabelUpdaterImage    = "IBM_VPC_NODE_LABEL_UPDATER_IMAGE"
+)
+
+func GetIBMVPCBlockCSIOperatorConfig() CSIOperatorConfig {
+	pairs := []string{
+		"${OPERATOR_IMAGE}", os.Getenv(envIBMVPCBlockDriverOperatorImage),
+		"${DRIVER_IMAGE}", os.Getenv(envIBMVPCBlockDriverImage),
+		"${NODE_LABEL_IMAGE}", os.Getenv(envIBMVPCNodeLabelUpdaterImage),
+	}
+
+	return CSIOperatorConfig{
+		CSIDriverName:   IBMVPCBlockCSIDriverName,
+		ConditionPrefix: "IBMVPCBlock",
+		Platform:        configv1.IBMCloudPlatformType,
```

Comment:
I need to come back to this. The operator is only for VPC clusters, but `IBMCloudPlatformType` doesn't distinguish between VPC and Classic. We'll have to look at the platform status and check for `IBMCloudProviderTypeVPC`:
https://github.com/openshift/api/blob/9b3afd95c93af476bec246b2c60f64235f078198/config/v1/types_infrastructure.go#L174-L187